lib/cdo/regexp.rb

```diff
@@ -16,6 +16,14 @@ def self.us_phone_number?(text)
     US_PHONE_NUMBER_ONLY_REGEXP === text
   end
 
```

Comment:
Good call. I switched to using a simple regex to make sure we have at least 10 digits (and not care about the format)